sdk/eventhub/event-hubs/test/stress/app/src/scenarioGetRuntimeProps.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { EventHubConsumerClient } from "@azure/event-hubs";
+import {
+  EventHubsStressTester,
+  createEventHubsProducerClient,
+  defaultClientAppInsights,
+} from "./eventHubsStressTester";
+import parsedArgs from "minimist";
+import { delay } from "@azure/core-amqp";
+
+interface scenarioGetRuntimePropertiesOptions {
+  testDurationInMs?: number;
+}
+
+function sanitizeOptions(args: string[]): Required<scenarioGetRuntimePropertiesOptions> {
+  const options = parsedArgs<scenarioGetRuntimePropertiesOptions>(args);
+  return {
+    testDurationInMs: options.testDurationInMs || 20 * 24 * 60 * 60 * 1000, // Default = 20 days
+  };
+}
+
+async function scenarioGetRuntimeProperties() {
+  const testOptions = sanitizeOptions(process.argv);
+  const { testDurationInMs } = testOptions;
+  const startedAt = new Date();
+
+  const producer = createEventHubsProducerClient();
+  const stressBase = new EventHubsStressTester({
+    testName: "GetRuntimeProperties-memLeak",
+    snapshotIntervalInMs: 1000,
+    writeSnapshotInfoToConsole: true,
+  });
+  const connectionString = process.env.EVENTHUBS_CONNECTION_STRING;
+  const eventHubName = process.env.EVENTHUB_NAME;
+
+  const consumerClient = new EventHubConsumerClient(
+    EventHubConsumerClient.defaultConsumerGroupName,
+    connectionString || "",
+    eventHubName || ""
```

Comment:
nitpick: I think we should error with a clear message if the env vars are not defined?